docusaurus/docs/dev-docs/configurations/database.md

```diff
@@ -44,40 +44,41 @@ The `./config/database.js` (or `./config/database.ts` for TypeScript) accepts 2
+| `connection`                                             | Database [connection information](#connection-parameters)                                                                                                                                                                                                                 | `Object`  | -       |
+| `debug`                                                  | Show database exchanges and errors.                                                                                                                                                                                                                                       | `Boolean` | `false` |
+| `useNullAsDefault`<br/><br />_Optional, only for SQLite_ | Use `NULL` as a default value                                                                                                                                                                                                                                             | `Boolean` | `true`  |
+| `pool`<br /><br />_Optional_                             | [Database pooling options](#database-pooling-options)                                                                                                                                                                                                                     | `Object`  | -       |
+| `acquireConnectionTimeout`<br /><br />_Optional_         | How long knex will wait before throwing a timeout error when acquiring a connection (in milliseconds)                                                                                                                                                                     | `Integer` | `60000` |
 
 :::note
 Strapi only supports the following client values, and will automatically rewrite the `client` value to the following options before passing the configuration to Knex:
 
-| `client` value | Actual package used                                             |
-|----------------|-----------------------------------------------------------------|
-| sqlite         | [better-sqlite3](https://www.npmjs.com/package/better-sqlite3)  |
-| mysql          | [mysql2](https://www.npmjs.com/package/mysql2)                  |
-| postgres       | [pg](https://www.npmjs.com/package/pg)                          |
+| `client` value         | Actual package used                                            |
+| ---------------------- | -------------------------------------------------------------- |
+| sqlite                 | [better-sqlite3](https://www.npmjs.com/package/better-sqlite3) |
+| mysql                  | [mysql2](https://www.npmjs.com/package/mysql2)                 |
+| postgres & cockroachdb | [pg](https://www.npmjs.com/package/pg)                         |
```

Comment:
I guess the value is either 'postgress' or 'cockroachdb', right?

So I'd duplicate the line line to avoid confusion and stick to what users actually need to type:

```suggestion
| postgres        | [pg](https://www.npmjs.com/package/pg)                     |
| cockroachdb | [pg](https://www.npmjs.com/package/pg)                     |  
```